build/charts/antrea/crds/flowexporterdestination.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: flowexporterdestinations.crd.antrea.io
+  labels:
+    app: antrea
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+      - name: Address:Port
+        type: string
+        description: Address of flow collector.
+        jsonPath: .spec.address
+      - name: Protocol
+        type: string
+        description: Protocol used to send network flows.
+        jsonPath: .spec.protocol
+      - name: Filter
+        type: string
+        description: Filter for network flows, nil means no filter.
+        jsonPath: .spec.filter
+        priority: 10
```

Comment:
ditto
I would actually remove this one, I don't think it's needed in the default `kubectl` output

Reply:
Yes, esp. if the column is just going to be a JSON object, but I don't feel very strongly about it